src/main/java/microsoft/exchange/webservices/data/HttpClientWebRequest.java

```diff
@@ -127,6 +138,14 @@ public void prepareConnection() throws EWSHttpException {
       rcBuilder.setConnectTimeout(getTimeout());
       rcBuilder.setRedirectsEnabled(isAllowAutoRedirect());
       rcBuilder.setSocketTimeout(getTimeout());
+	  
+	  // fix issue #144 + #160: if we used NTCredentials from above: these are NT credentials
+      if (getUserName() != null) {
+    	ArrayList<String> authPrefs = new ArrayList<String>();
+    	authPrefs.add(AuthSchemes.NTLM);
+    	rcBuilder.setTargetPreferredAuthSchemes(authPrefs).build();
+      }
+      //
```

Comment:
- prepareAsyncConnection: I assule so however I haven't verified this.
- build() => fixed.